Header header

string setPerformative
string sender
string receiver
string content
string language
string ontology